tests/runahead/no_final/suite.rc

```diff
@@ -17,4 +17,6 @@
     [[bar]]
         script = true
     [[shutdown]]
-        script = cylc shutdown $CYLC_SUITE_REG_NAME
+        script = """
+sleep 5
+cylc shutdown $CYLC_SUITE_REG_NAME"""
```

Comment:
The shutdown command was taking effect before the started message was processed from this task.  Due to backgrounding the started message in the job script?  This could be a reason for recent test instability?  If started message and task content are happening concurrently, the order of associated events will probably be less consistent.